lib/sql_dust/utils/path_utils.ex

```diff
@@ -33,19 +34,19 @@ require IEx
   def prepend_path_aliases2(sql, options) when is_binary(sql) do
```

Comment:
Can you rename these I know my fault with my modifications but was more proof of concept on how to get more performance.